packages/prisma/migrations/20241210122559_support_event_level_selected_calendar/migration.sql

```diff
@@ -0,0 +1,31 @@
+/*
+  Warnings:
+
+  - The primary key for the `SelectedCalendar` table will be changed. If it partially fails, the table could be left without primary key constraint.
+  - A unique constraint covering the columns `[userId,integration,externalId,eventTypeId]` on the table `SelectedCalendar` will be added. If there are existing duplicate values, this will fail.
+  - A unique constraint covering the columns `[googleChannelId,eventTypeId]` on the table `SelectedCalendar` will be added. If there are existing duplicate values, this will fail.
+
+*/
+-- DropIndex
+DROP INDEX "SelectedCalendar_googleChannelId_key";
+
+-- AlterTable
+ALTER TABLE "EventType" ADD COLUMN     "useEventLevelSelectedCalendars" BOOLEAN NOT NULL DEFAULT false;
+
+-- AlterTable
+ALTER TABLE "SelectedCalendar" DROP CONSTRAINT "SelectedCalendar_pkey",
+ADD COLUMN     "eventTypeId" INTEGER,
+ADD COLUMN     "id" SERIAL NOT NULL,
+ADD CONSTRAINT "SelectedCalendar_pkey" PRIMARY KEY ("id");
+
+-- CreateIndex
+CREATE INDEX "SelectedCalendar_eventTypeId_idx" ON "SelectedCalendar"("eventTypeId");
+
+-- CreateIndex
+CREATE UNIQUE INDEX "SelectedCalendar_userId_integration_externalId_eventTypeId_key" ON "SelectedCalendar"("userId", "integration", "externalId", "eventTypeId");
+
+-- CreateIndex
+CREATE UNIQUE INDEX "SelectedCalendar_googleChannelId_eventTypeId_key" ON "SelectedCalendar"("googleChannelId", "eventTypeId");
```

Comment:
The unique constraint on googleChannelId is also changing to composite key with eventTypeId .